openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -0,0 +1,2038 @@
+    });
+
+    // Set the thematic layer options
+    var options = {
+        recordsField: 'features',
+        locationMode: L.LocationModes.GEOJSON,
+        geoJSONField: 'geometry',
+        layerOptions: {
+            fillOpacity: 0.5,
+            opacity: 0.5,
+            weight: 1,
+            stroke: true,
+            color: '#0000FF'
+        },
+        displayOptions: {}
+    };
+
+    options.displayOptions[displayElement] = {
+        displayName: selectedIndicator,
+        fillColor: yellowToRed
+    };
+
+    thematicLayer = new L.ChoroplethDataLayer(layerAttributes, options);
+    map.addLayer(thematicLayer);
+
+    legendControl = new L.Control.Legend();
+    legendControl.addTo(map);
+    $('#absoluteSpinner').hide();
+}
+
+function whenProjDefSelected() {
+    projDefChange = true;
+    $('#projectDef-spinner').show();
+    var pdSelection = $('#pdSelection').val();
+    for (var i = 0; i < tempProjectDef.length; i++) {
+        if (tempProjectDef[i].title === pdSelection) {
+            // Deep copy the temp project definition object
+            sessionProjectDef = jQuery.extend(true, {}, tempProjectDef[i]);
+            if (has_custom_fields(sessionProjectDef)) {
+                projDefOperatorsAreStandard = false;
+                $('.alert-custom-operators').show();
+            } else {
+                projDefOperatorsAreStandard = true;
+                $('.alert-custom-operators').hide();
+            }
+            loadPD(sessionProjectDef);
+            $('#iri-spinner').hide();
+            $('#project-definition-svg').show();
+            processIndicators(layerAttributes, sessionProjectDef);
+        }
+    }
+}
+
+function has_custom_fields(projDef) {
+    if (typeof(projDef.operator) !== 'undefined' && namesToOperators[projDef.operator].code == 'CUSTOM') {
+        return true;
+    }
+    for (var childIdx in projDef.children) {
+        var child = projDef.children[childIdx];
+        if (has_custom_fields(child)) {
+            return true;
+        }
+    }
+    return false;
+}
+
+function getGeoServerLayers() {
+    $('#load-project-spinner').show();
+    var IRMTLayerNames = [];
+    var IRMTLayerTitle = [];
+    var url = "http://"+ window.location.hostname +":8000/geoserver/ows?service=WFS&version=1.0.0&REQUEST=GetCapabilities&SRSNAME=EPSG:4326&outputFormat=json&format_options=callback:getJson";
+    // Get layers from GeoServer and populate the layer selection menu
+
+    $.ajax({
+        url: url,
+        contentType: 'application/json',
+        success: function(xml) {
+
+
+            //convert XML to JSON
+            var xmlText = new XMLSerializer().serializeToString(xml);
+            var x2js = new X2JS();
+
+            var jsonElement = x2js.xml_str2json(xmlText);
+
+            var featureType = jsonElement.WFS_Capabilities.FeatureTypeList.FeatureType;
+
+            // Find the IRMT keywords
+            var stringsToLookFor = ['SVIR', 'IRMT_QGIS_Plugin'];
```

Comment:
Changed this.
Before was: 
var stringsToLookFor = ['SVIR_QGIS_Plugin', 'IRMT_QGIS_Plugin'];

Reply:
I suppose it was safer as it was before. Why did you change this?